descriptors/postgres/debezium-postgres-1.9.0.Alpha1.json

```diff
@@ -290,284 +154,13 @@
-          "x-category" : "CONNECTOR"
-        },
-        "decimal.handling.mode" : {
-          "title" : "Decimal Handling",
-          "description" : "Specify how DECIMAL and NUMERIC columns should be represented in change events, including:'precise' (the default) uses java.math.BigDecimal to represent values, which are encoded in the change events using a binary representation and Kafka Connect's 'org.apache.kafka.connect.data.Decimal' type; 'string' uses string to represent values; 'double' represents values using Java's 'double', which may not offer the precision but will be far easier to use in consumers.",
-          "default" : "precise",
-          "enum" : [ "string", "double", "precise" ],
-          "type" : "string",
-          "x-name" : "decimal.handling.mode",
-          "x-category" : "CONNECTOR"
-        },
-        "binary.handling.mode" : {
-          "title" : "Binary Handling",
-          "description" : "Specify how binary (blob, binary, etc.) columns should be represented in change events, including:'bytes' represents binary data as byte array (default)'base64' represents binary data as base64-encoded string'hex' represents binary data as hex-encoded (base16) string",
-          "default" : "bytes",
-          "enum" : [ "bytes", "base64", "hex" ],
-          "type" : "string",
-          "x-name" : "binary.handling.mode",
-          "x-category" : "CONNECTOR"
-        },
-        "time.precision.mode" : {
-          "title" : "Time Precision",
-          "description" : "Time, date, and timestamps can be represented with different kinds of precisions, including:'adaptive' (the default) bases the precision of time, date, and timestamp values on the database column's precision; 'adaptive_time_microseconds' like 'adaptive' mode, but TIME fields always use microseconds precision;'connect' always represents time, date, and timestamp values using Kafka Connect's built-in representations for Time, Date, and Timestamp, which uses millisecond precision regardless of the database columns' precision .",
-          "default" : "adaptive",
-          "enum" : [ "adaptive", "adaptive_time_microseconds", "connect" ],
-          "type" : "string",
-          "x-name" : "time.precision.mode",
-          "x-category" : "CONNECTOR"
-        },
-        "include.schema.comments" : {
-          "title" : "Include Table and Column Comments",
-          "description" : "Whether the connector parse table and column's comment to metadata object.Note: Enable this option will bring the implications on memory usage. The number and size of ColumnImpl objects is what largely impacts how much memory is consumed by the Debezium connectors, and adding a String to each of them can potentially be quite heavy. The default is 'false'.",
-          "default" : false,
-          "type" : "boolean",
-          "x-name" : "include.schema.comments",
-          "x-category" : "CONNECTOR"
-        },
-        "interval.handling.mode" : {
```

Comment:
@gunnarmorling I would leave `interval.handling.mode` as excluded, but should the Debezium Operator maybe set the value to "string"?

Reply:
Ok, makes sense. We should then set the default to string for Debezium 2.0 in general though. Can you add this to the DBZ 2.0 issue?